xbmc/epg/GUIEPGGridContainerModel.cpp

```diff
@@ -258,8 +261,9 @@ void CGUIEPGGridContainerModel::FindChannelAndBlockIndex(int channelUid, unsigne
     CDateTime gridCursor(m_gridStart); //reset cursor for new channel
     unsigned long progIdx = m_epgItemsPtr[channel].start;
     unsigned long lastIdx = m_epgItemsPtr[channel].stop;
-    int iEpgId            = m_programmeItems[progIdx]->GetEPGInfoTag()->EpgID();
+    int iEpgId = m_programmeItems[progIdx]->GetEPGInfoTag()->EpgID();
     CEpgInfoTagPtr tag;
+    CPVRChannelPtr chan;
```

Comment:
channel is already taken for another variable. no const, because used inside for loop and reassigned while iterating.